services/order.py

```diff
@@ -0,0 +1,40 @@
+from django.utils import timezone
+from db.models import User, Ticket, Order, MovieSession
+from typing import Optional
+from django.db import transaction
+
+
+def create_order(*, username, tickets, date=None):
+
+    with transaction.atomic():
+        user = User.objects.get(username=username)
+
+        # Якщо дата не передана, використовуємо поточний час
+        if not date:
+            date = timezone.now()
+
+        # Створення замовлення
+        order = Order.objects.create(user=user, created_at=date)
+
+        # Створення квитків для цього замовлення
+        for ticket in tickets:
+            movie_session = MovieSession.objects.get(id=ticket['movie_session'])
+            Ticket.objects.create(
+                order=order,
+                row=ticket['row'],
+                seat=ticket['seat'],
+                movie_session=movie_session,
+
+            )
```

Comment:
Consider validating the `tickets` input to ensure each ticket dictionary contains the necessary keys ('movie_session', 'row', 'seat') and that the values are valid. This will prevent runtime errors during ticket creation.